src/Type/Doctrine/Descriptors/Ramsey/UuidBinaryOrderedTimeTypeDescriptor.php

```diff
@@ -0,0 +1,32 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Doctrine\Descriptors\Ramsey;
+
+use PHPStan\Type\Doctrine\Descriptors\DoctrineTypeDescriptor;
+use PHPStan\Type\Type;
+use PHPStan\Type\TypeCombinator;
+use Ramsey\Uuid\Doctrine\UuidBinaryOrderedTimeType;
+use Ramsey\Uuid\UuidInterface;
+
+class UuidBinaryOrderedTimeTypeDescriptor implements DoctrineTypeDescriptor
+{
+
+	public function getType(): string
```

Comment:
All classes are identical except for the `getType()` method

Reply:
Instead, inject the type into the constructor and register the same class as three different services :)

Also a test case would be nice, you can add one of the descriptors to EntityColumnRuleTest and add an entity field with UUID type there :)